cloud/auth/auth.go

```diff
@@ -395,9 +394,8 @@ func Login(domain, orgID string, client astro.Client, out io.Writer, shouldDispl
 		fmt.Println("You are logging into Astro via an OAuth token\nThis token will expire in 24 hours and will not refresh")
 		fmt.Printf("\nPlease visit the following URL, authenticate and paste token in next prompt\n")
 		fmt.Println("cloud." + domain + "/token\n")
```

Comment:
is this still relevant?

Reply:
Maybe not, because they already have the token. Let me see if i can add a reference in the help menu for the same.